### ISSUE TITLE
Update activesupport gem to version 5.2.4.3, 6.0.3.1 or higher

### ISSUE THREAD

**Issue by DominikSecka** [CocoaPods/CocoaPods#9925]
* [x] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

SNYK validation

## What did you expect to happen?

activesupport gem need to be updated to solve this issue -> https://app.snyk.io/vuln/SNYK-RUBY-ACTIVESUPPORT-569598

## What happened instead?

activesupport gem security issue was reported -> https://app.snyk.io/vuln/SNYK-RUBY-ACTIVESUPPORT-569598

### Stack

```
   CocoaPods : 1.9.3
        Ruby : ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19]
    RubyGems : 3.0.3
        Host : Mac OS X 10.15.5 (19F101)
       Xcode : 11.5 (11E608c)
         Git : git version 2.27.0
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib
Repositories : trunk - CDN - https://cdn.cocoapods.org/
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-check       : 1.1.0
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.5.0
cocoapods-try         : 1.2.0
```

### Podfile

```ruby
#source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '11.0'
use_frameworks!
# ignore all warnings from all pods
inhibit_all_warnings!

#workspace <wokrspace>

def app_pods
    ...
end

target <target> do
    app_pods

    target <target> do
        inherit! :search_paths
        ...
    end
end
```

## Project that demonstrates the issue
it's only an issue with CocoaPods the tool

**Comment by dnkoutso** [comment]
Version 1.10.0.beta.1 was just shipped last week with support > 5. @amorde interestingly, CocoaPods gem is >5 but Core is >5 but < 6. Any reason we shouldnt match them?

**Comment by amorde** [comment]
That difference was unintentional - we should match them, I don't think we need `< 6` on Core

**Comment by dnkoutso** [comment]
We cannot update to remove `< 6` because we support Ruby 2.3.3 as minimum.

@DominikSecka your issue title still says 5.2.4.3 which our 1.10.0.beta.1 release should support here. is this issue fixed for 1.10.0?
